src/Symfony/Component/Routing/Loader/AnnotationClassLoader.php

```diff
@@ -12,7 +12,12 @@
 namespace Symfony\Component\Routing\Loader;
 
 use Doctrine\Common\Annotations\Reader;
+use InvalidArgumentException;
+use LogicException;
+use ReflectionClass;
+use ReflectionMethod;
```

Comment:
Symfony never add use statements for classes of the global namespace. So please revert these changes